natgeo/mortar
# [AEM-6034] Updates to make lates/most read component more AEM friendly

https://jira.natgeo.com/browse/AEM-6034

To test:

- Pull AEM-6034
- Run `npm test` to verify tests/linting pass
- Run `gulp serve`
- Navigate to http://localhost:3000/contentPackages/contentPackage/index.html
- Verify layout

These changes are needed for the implementation into AEM, and need to merge first so AEM can inherit the newest code. This is the PR for AEM https://github.com/natgeo/aem-all/pull/1695
